tsnet/tsnet.go

```diff
@@ -908,6 +908,34 @@ func (s *Server) Listen(network, addr string) (net.Listener, error) {
 	return s.listen(network, addr, listenOnTailnet)
 }
 
+// ListenPacket announces on the Tailscale network.
+//
+// The network must be "udp", "udp4" or "udp6". The addr must be of the form
+// "ip:port" where ip is a valid IPv4 or IPv6 address corresponding to "udp4" or
+// "udp6" respectively. IP must be specified.
```

Comment:
So this does seem to work, though requires the IP to be specified makes it a bit annoying to work with.  Having this call s.Start() is great and all, but in order to have the TailscaleIP, you would have already had to have the server running and connected.  Could this call `s.Up` in the same way that `ListenTLS` does, and grab the IP address automatically?

Reply:
It does occur to me that I could also just listen on `0.0.0.0` which seems to work fine, but I still think accepting no address is a little nicer.

Reply:
`0.0.0.0` in netstack doesn't actually mean bind on all interfaces, it specifically listens on `0.0.0.0` :)